src/apisix/plugins/bk-permission.lua

```diff
@@ -72,31 +71,34 @@ function _M.access(conf, ctx)
         return
     end
 
+    local app_code = ctx.var.bk_app_code
+
+    if pl_types.is_empty(app_code) then
+        return errorx.exit_with_apigw_err(
+            ctx, errorx.new_app_no_permission():with_fields(
+                {
+                    reason = "no permission, app is not verified, please provide valid app info to verify app.",
```

Comment:
please contact the administrator of this gateway. The reason for the error here is that: if resource_perm_required is true, verified_app_required must be set to true too.